test/Altinn.App.Api.Tests/Altinn.App.Api.Tests.csproj

```diff
@@ -5,7 +5,7 @@
 		<Nullable>enable</Nullable>
 		<ImplicitUsings>enable</ImplicitUsings>
 		<IsPackable>false</IsPackable>
-		<NoWarn>$(NoWarn);CS1591;CS0618</NoWarn>
+		<NoWarn>$(NoWarn);CS1591;CS0618;CS7022</NoWarn>
 		<!--
 			CS1591: Don't warn for missing XML doc
 			CS0618: This is a test project, so we usually continue testing [Obsolete] apis
```

Comment:
CS7022: The entry point of the program is global code; ignoring 'Program.Main(string[])' entry point

Reply:
```suggestion
			CS0618: This is a test project, so we usually continue testing [Obsolete] apis
			CS7022: The entry point of the program is global code; ignoring 'Program.Main(string[])' entry point
```